src/ProcessPool.php

```diff
@@ -175,7 +174,7 @@ protected function start()
      */
     protected function createProcess()
     {
-        return new WorkerProcess((new Process(
+        return new WorkerProcess((new BackgroundProcess(
```

Comment:
This prevents the supervisor to send SIGKILL signal to worker on object distruction. It may also be a good idea to send SIGTERM in the BackgroudProcess::__destruct().